README.rst

```diff
@@ -45,13 +45,13 @@ An incomplete list:
 * `OpenML-Python <https://openml.github.io/openml-python/main/examples/index.html>`_
 * `Optuna <https://optuna.readthedocs.io/en/stable/tutorial/index.html>`_
 * `PlasmaPy <https://docs.plasmapy.org/en/latest/examples.html>`_
-* `pyRiemann <https://pyriemann.readthedocs.io/en/latest/index.html>`_
+* `PyRiemann <https://pyriemann.readthedocs.io/en/latest/index.html>`_
```

Comment:
looking at the website it's `pyRiemann`:

![Screenshot from 2023-03-13 12-50-43](https://user-images.githubusercontent.com/2365790/224771328-941500f2-966a-4444-b557-d5e0712312b1.png)

I'm inclined to move in the other direction by respecting each project's preferred capitalization

Reply:
> respecting each project's preferred capitalization

... this would imply the following changes:

Auto-sklearn --> auto-sklearn
Fury --> FURY
GIMLY --> pyGIMLi
MNE-python --> MNE-Python
OpenML-Python --> OpenML
Radis --> RADIS
Scikit-learn --> scikit-learn

shall I add these changes to this PR?

Reply:
Yes that would be good!

Reply:
done